tests/query.rs

```diff
@@ -0,0 +1,121 @@
+// Copyright 2016 Mozilla
+//
+// Licensed under the Apache License, Version 2.0 (the "License"); you may not use
+// this file except in compliance with the License. You may obtain a copy of the
+// License at http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software distributed
+// under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+// CONDITIONS OF ANY KIND, either express or implied. See the License for the
+// specific language governing permissions and limitations under the License.
+
+extern crate time;
+
+extern crate mentat;
+extern crate mentat_core;
+extern crate mentat_db;
+
+use mentat_core::{
+    TypedValue,
+    ValueType,
+};
+
+use mentat::{
+    NamespacedKeyword,
+    QueryResults,
+    new_connection,
+    q_once,
+};
+
+#[test]
+fn test_query() {
```

Comment:
nit: consider splitting this test function into individual bits, so that the failure messages are clearer.